.github/workflows/publish.yml

```
@@ -47,3 +47,11 @@ jobs:
         with:
           user: __token__
           password: ${{ secrets.PYPI_API_TOKEN }}
+      - name: Create Release Notes
```

Totally not needed, and maybe "Release Notes" is the standard, but otherwise this is more consistent with above
```suggestion
      - name: Create release notes
```

---

Oops, copied/pasted this and didn't notice the inconsistency.